src/site/stages/build/index.js

```diff
@@ -15,29 +15,30 @@ const getOptions = require('./options');
 const registerLiquidFilters = require('../../filters/liquid');
 const { getDrupalContent } = require('./drupal/metalsmith-drupal');
 const addDrupalPrefix = require('./plugins/add-drupal-prefix');
-const createBuildSettings = require('./plugins/create-build-settings');
```

Comment:
Looks like a lot of edits here but just alphabetized the plugins imports because I though it'd be easier to read.